nf-core-contributors.yaml

```diff
@@ -673,3 +673,21 @@ contributors:
       contact_github: lbarquist
       location: [49.801900, 9.957067]
       twitter: Helmholtz_HIRI
+
+    - full_name: Institute of Zoology
+      short_name: IOZ
+      description: >
+        Institute of Zoology (IOZ), Chinese Academy of Sciences (CAS),
+        is a government-funded research institution in zoological sciences.
+        It has a long history of over 80 years.
+        The predecessor of IOZ was Fan Memorial Institute of Biology founded in 1928.
+      address: Institute of Zoology, Chinese Academy of Sciences (CAS), Beijing, 100101, China
+      url: http://english.ioz.cas.cn/
+      affiliation: Chinese Academy of Sciences
+      affiliation_url: http://english.cas.cn/
+      image_fn: IOZ.svg
+      contact: atongsa
```

Comment:
Can you put in a full name here please? Thank you :-)

Reply:
ok, i will , thank you